### PR TITLE
Shoe Snatching Charm

### PR DESCRIPTION
**Pull request by Arthurdentist**
Adds a new spell, the Shoe Snatching Charm.

This spell allows you to steal your victims shoes from a distance. It has the same range as the Horsemask spell for comparison. So long as you can see your target you can steal his shoes basically. This does mean that if you have the x-ray ability you can steal shoes through walls but I consider that fine and funny as fuck anyway.

Also changed the incorrect description of the wizard car in the spell book (RIP Jaunt)

I don't consider this finished but I am putting this out there so that people can see it.

### CODE REVIEW

**Comment by DrCelt** [comment]
my god yes

**Comment by Arthurdentist** [comment]
Things to note: Everything works fine but the only thing that bothers me is the fact that the spell produces sparks even when the victim isn't wearing any shoes. Im not sure how to prevent this.

Also the HUD icon is a picture of the Galoshes, nicest looking shoe icon I thought.

**Comment by Probe1** [comment]
Willy would be proud.  Sounds really good.  More creative spells less murderfist.

File: code/modules/spells/targeted/shoesnatch.dm [hunk 1]
```diff
@@ -0,0 +1,32 @@
+/spell/targeted/shoesnatch
+
+	name = "Shoe Snatching Charm"
+	desc = "This spell will allow you to steal somebodies shoes right off of their feet!"
+	school = "evocation"
+	charge_type = Sp_RECHARGE
+	charge_max = 150
+	charge_counter = 0
```
Comment (ComicIronic) [inline code/modules/spells/targeted/shoesnatch.dm]: No need to set this.

File: code/game/gamemodes/wizard/spellbook.dm [hunk 2]
```diff
@@ -71,13 +71,15 @@
 			<I>This spell will curse a person to wear an unremovable horse mask (it has glue on the inside) and speak like a horse. It does not require wizard garb.</I><BR>
 			<A href='byond://?src=\ref[src];spell_choice=frenchcurse'>The French Curse</A> (30)<BR>
 			<I>This spell silences sombody adjacent to you, and curses them with an unremovable Mime costume.</I><BR>
+			<A href='byond://?src=\ref[src];spell_choice=shoesnatch'>Shoe Snatching Charm</A> (15)<BR>
+			<I>This spell will remove your victim's shoes and materialize them in your hands.</I><BR>
```
Comment (ComicIronic) [inline code/game/gamemodes/wizard/spellbook.dm]: You should mention the lack of requirement for clothes.

File: code/modules/spells/targeted/shoesnatch.dm [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+	..()
+	for(var/mob/living/carbon/human/target in targets)
+		var /obj/old_shoes = target.shoes
+		if(old_shoes)
+			sparks_spread = 1
+			sparks_amt = 4
+			target.drop_from_inventory(old_shoes)
+			target.visible_message(	"<span class='danger'>[target]'s shoes suddenly vanish!</span>", \
+									"<span class='danger'>Your shoes suddenly vanish!</span>")
+			usr.put_in_active_hand(old_shoes)
```
Comment (ComicIronic) [inline code/modules/spells/targeted/shoesnatch.dm]: user, not usr.

File: code/modules/spells/targeted/shoesnatch.dm (hunk 1 above)
Comment (Arthurdentist) [inline code/modules/spells/targeted/shoesnatch.dm]: What doesn't need setting specifically?

Comment (ComicIronic) [inline code/modules/spells/targeted/shoesnatch.dm]: charge_counter. It is set to max when the spell is generated anyways, so setting it to 0 in the definition is pointless.

**Comment by Kurfursten** [comment]
:+1: If I didn't approve of it, I wouldn't have helped you bugfix it, but it doesn't hurt to make the support official

File: code/game/gamemodes/wizard/spellbook.dm [hunk 4]
```diff
@@ -71,13 +71,15 @@
 			<I>This spell will curse a person to wear an unremovable horse mask (it has glue on the inside) and speak like a horse. It does not require wizard garb.</I><BR>
 			<A href='byond://?src=\ref[src];spell_choice=frenchcurse'>The French Curse</A> (30)<BR>
 			<I>This spell silences sombody adjacent to you, and curses them with an unremovable Mime costume.</I><BR>
+			<A href='byond://?src=\ref[src];spell_choice=shoesnatch'>Shoe Snatching Charm</A> (15)<BR>
+			<I>This spell will remove your victim's shoes and materialize them in your hands. This spell does not recquire robes to cast.</I><BR>
```
Comment (ComicIronic) [inline code/game/gamemodes/wizard/spellbook.dm]: RECQUIRE?

Comment (Arthurdentist) [inline code/game/gamemodes/wizard/spellbook.dm]: bluh

**Comment by IratePirate** [comment]
>wizard with fire extinguisher spraying glass shards everywhere
>Sec force rolling about on the floor with glass in their feet screaming for help

Nice.

**Pull request merged**